src/components/data/PathListAutomation.js

```diff
@@ -213,229 +219,252 @@ export default function PathListAutomation(props) {
+                                <Grid>
+                                    <Button
                                         id={buildId(
                                             baseId,
-
-                                            ids.PATH_LIST_AUTO_FILE_NAME_FIELD
+                                            ids.PATH_LIST_AUTO_DONE_BTN
                                         )}
-                                        name="fileName"
-                                        required={true}
-                                        label={t("fileName")}
-                                        component={SaveAsField}
-                                    />
-                                </Paper>
-                            </Grid>
-                        </Grid>
-                        <Grid
-                            container
-                            direction="row"
-                            justify="flex-end"
-                            alignItems="flex-end"
-                            spacing={1}
-                        >
-                            {createPathListError && (
-                                <Grid item xs>
-                                    <ErrorTypographyWithDialog
-                                        baseId={baseId}
-                                        errorMessage={errorMsg}
-                                        errorObject={createPathListError}
-                                    />
+                                        color="primary"
+                                        type="submit"
+                                        onClick={handleSubmit}
+                                    >
+                                        {i18nCommon("done")}
+                                    </Button>
                                 </Grid>
-                            )}
-                            <Grid item>
-                                <Button
-                                    id={buildId(
-                                        baseId,
-                                        ids.PATH_LIST_AUTO_CANCEL_BTN
-                                    )}
-                                    onClick={onCancel}
-                                >
-                                    {i18nCommon("cancel")}
-                                </Button>
-                            </Grid>
-                            <Grid item>
-                                <Button
-                                    id={buildId(
-                                        baseId,
-                                        ids.PATH_LIST_AUTO_DONE_BTN
-                                    )}
-                                    color="primary"
-                                    type="submit"
-                                >
-                                    {i18nCommon("done")}
-                                </Button>
-                            </Grid>
-                        </Grid>
```

Comment:
When you add back in the `Grid` portion from above, you'll also need to add this closing tag